cumulus/bridges/bin/runtime-common/src/integrity.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// Copyright Parity Technologies (UK) Ltd.
```

Comment:
should it be Copyright (C) for consistency with polkadot changes? 
https://github.com/paritytech/polkadot/pull/7034

Reply:
Makes sense to me, added `(C)` for consistency.